### DISCUSSION TITLE
Connected Git Repository Error: Project Link not found

### DISCUSSION THREAD

**Discussion by briankabugi** [vercel/community#6491] [Account Issue] [answered]
I'm having trouble updating my vercel github URL since the page no longer exists. All I'm seeing is an error as below ; Connected Git Repository Error: Project Link not found.

<img width="668" alt="Screenshot 2024-04-13 121658" src="https://github.com/vercel/community/assets/166870438/7429afc2-1c2d-446c-a1bd-cdffcc056fef">

### Question

Anyone encountered this before and found a solution? How to update github url if this error pops up 

#vercel #change-github-account-in-vercelproject @vercel

**Comment by amyegan** [comment, marked as answer]
That [lost git repository access](https://vercel.com/docs/errors/error-list#lost-git-repository-access) error alert appears when a project is unable to connect to its configured git repo. If you click the **Reconnect** button, that should remove the old connection and give you the opportunity to add a new one.

**Reply by briankabugi** [reply]
Thank you very much @amyegan

**Reply by dellsony7** [reply]
I reconnected the repo, but it didn't updated the changes I'm doing and in my repo it doesn't appear as the deployment was done

**Reply by amyegan** [reply]
It could be a project configuration issue. Do you see an error if you deploy from Vercel CLI?

https://vercel.com/docs/cli/deploying-from-cli

[a comment by an automated account is not shown]
